src/challenges.js

```diff
@@ -1,16 +1,31 @@
 /* ======================== CallBacks Practice ============================ */
 const each = (elements, cb) => {
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);// passes each element in arr to cb
```

Comment:
Comments are always helpful for the next person or the person reviewing the code, Good job Alexis!